interfaces/matlab/toolbox/@ThermoPhase/elementMassFraction.m

```diff
@@ -0,0 +1,46 @@
+function elMassFrac = elementMassFraction(tp, element)
+% ELEMENTMASSFRACTION  Determine the elemental mass fraction in gas object.
+% elMassFrac = elementMassFraction(tp, element)
+% :param tp:
+%     Object representing the gas, instance of class :mat:func:`Solution`, 
+%     and an ideal gas. The state of this object should be set to an 
+%     estimate of the gas state before calling elementMassFraction.
+% :param element:
+%     String representing the element name. 
+% :return:
+%     Elemental mass fraction within a gas object.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Check input parameters
```

Comment:
I think you need a blank line at the end of the method docstring so that this comment doesn't end up as part of it (at least as read by the Sphinx converter)